sycl/test/hier_par/hier_par_wgscope.cpp

```diff
@@ -14,9 +14,10 @@
 
 // RUN: %clangxx -O0 -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
 // RUN: env SYCL_DEVICE_TYPE=HOST %t.out
-// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// TODO re-enable CPU and ACC(uses CPU)  once CPU OpenCL RT in CI is updated:
```

Comment:
I suggest disable it using "XFAIL", so when it passes, we will notice it immediately.

Reply:
The reason why I did not use XFAIL is because in that case tests w/o -O0, which pass on CPU and ACC, won't be checked. To avoid that I'd need to duplicate the test.

But OK, AFAIK, drivers will be updated soon, so maybe that's not a big problem then.